### PR TITLE
Fix model update test

### PR DESCRIPTION
**Pull request by douglasselias**
@rafiss @RichardJCai 

One of the tests fails because it returns a string when doing a query instead of an integer, this is a expected behavior from cockroachDB?

### CODE REVIEW

File: tests/model_update_test.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+require('./helper');
+
+var expect = require('chai').expect;
+var Sequelize = require('..');
+
+const DataTypes = Sequelize.DataTypes;
+const sinon = require('sinon');
```
Comment (RichardJCai) [inline tests/model_update_test.js:7]: sinon isn't added as a dep in package.json

**Comment by RichardJCai** [comment]
Also it looks like model/update is still failing in CI

```
  [POSTGRES] Model
    update
      ✓ should only update the passed fields
      ✓ changed should be false after reload
      1) should ignore undefined values without throwing not null validation
      2) should return the updated record
      skips update query
        ✓ if no data to update
        ✓ skips when timestamps disabled


  4 passing (692ms)
  2 failing

  1) [POSTGRES] Model
       update
         should ignore undefined values without throwing not null validation:
     AssertionError: expected '2' to equal 2
      at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:1031:12)
      at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
      at doAsserterAsyncAndAddThen (node_modules/chai-as-promised/lib/chai-as-promised.js:289:22)
      at Proxy.<anonymous> (node_modules/chai-as-promised/lib/chai-as-promised.js:255:20)
      at Proxy.overwritingMethodWrapper (node_modules/chai/lib/chai/utils/overwriteMethod.js:78:33)
      at /home/runner/work/sequelize-cockroachdb/sequelize-cockroachdb/.downloaded-sequelize/test/integration/model/update.test.js:121:39
  From previous event:
      at Context.<anonymous> (test/integration/model/update.test.js:120:10)
  From previous event:
      at processImmediate (internal/timers.js:461:21)

  2) [POSTGRES] Model
       update
         should return the updated record:
     AssertionError: expected '2' to equal 2
      at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:1031:12)
      at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
      at doAsserterAsyncAndAddThen (node_modules/chai-as-promised/lib/chai-as-promised.js:289:22)
      at Proxy.<anonymous> (node_modules/chai-as-promised/lib/chai-as-promised.js:255:20)
      at Proxy.overwritingMethodWrapper (node_modules/chai/lib/chai/utils/overwriteMethod.js:78:33)
      at /home/runner/work/sequelize-cockroachdb/sequelize-cockroachdb/.downloaded-sequelize/test/integration/model/update.test.js:135:44
  From previous event:
      at /home/runner/work/sequelize-cockroachdb/sequelize-cockroachdb/.downloaded-sequelize/test/integration/model/update.test.js:133:14
  From previous event:
      at Context.<anonymous> (test/integration/model/update.test.js:127:52)
  From previous event:
      at processImmediate (internal/timers.js:461:21)
      ```

**Comment by RichardJCai** [comment]
As discussed before, it's failing because it's returning a string instead of an int and that will depend on https://github.com/cockroachdb/sequelize-cockroachdb/issues/50

I'd be okay with merging this first though.

File: tests/model_update_test.js (hunk 1 above)
Comment (douglasselias) [reply tests/model_update_test.js:7]: seems like I forgot, I will add then

**Review by RichardJCai — APPROVED** [review]

**Comment by RichardJCai** [comment]
LGTM barring merge conflict

**Pull request merged**